h/streamer/conf/nginx.conf

```diff
@@ -34,9 +33,7 @@ http {
     server_name _;
     server_tokens off;
 
-    root /var/www;
```

Comment:
We don't serve anything from `/var/www`. I think this is default config and just causes big confusion.